calculate_average_faridtmammadov.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
Please make this file executable.

Reply:
updated this file with 'git update-index --chmod=+x'